backend/apps/candidates/models.py

```diff
@@ -0,0 +1,145 @@
+    additional_contacts : str
+        additional contact info of the candidate
+    age : int
+        age of the candidate
+    updated_at : date
+        last time Candidate was updated
+    created_at : date
+        date of creation of the Candidate
+
+    """
+
+    class GenderChoices(models.TextChoices):
+        """This class provides enum for gender types."""
+
+        MALE = "M", _("Male")
+        FEMALE = "F", _("Female")
+        OTHER = "O", _("Other")
+
+    class EnglishLevelChoices(models.TextChoices):
+        """This class provides enum for levels of English."""
+
+        UNKNOWN = "A0", _("Unknown")
+        BEGINNER = "A1", _("Beginner")
+        ELEMENTARY = "A2", _("Elementary")
+        INTERMEDIATE = "B1", _("Intermediate")
+        UPPER_INTERMEDIATE = "B2", _("Upper-intermediate")
+        ADVANCED = "C1", _("Advanced")
+        PROFICIENCY = "C2", _("Proficiency")
+
+    class Meta:
+        """Standard Meta class for Candidate model."""
+
+        ordering = ("surname", "name")
+
+    name = models.CharField(
+        _("Firstname"),
+        max_length=50,
+        db_index=True,
+    )
+    surname = models.CharField(
+        _("Surname"),
+        max_length=50,
+        db_index=True,
+    )
+    date_of_birth = models.DateField(_("Birth date"), blank=True, null=True)
+    gender = models.CharField(
+        verbose_name=_("Gender"),
+        max_length=1,
+        choices=GenderChoices.choices,
+        default=GenderChoices.OTHER,
+    )
+    phone_number = PhoneNumberField(
+        verbose_name=_("Phone number"),
+        unique=True,
+        db_index=True,
+    )
+    email = models.EmailField(
+        _("Email address"),
+        max_length=100,
+        blank=True,
+        db_index=True,
+    )
+    level_of_english = models.CharField(
+        verbose_name=_("Level of English"),
+        max_length=2,
+        choices=EnglishLevelChoices.choices,
+        default=EnglishLevelChoices.UNKNOWN,
+    )
+    notes = models.TextField(
+        _("Additional info"),
+        max_length=1200,
+        blank=True,
+        null=True,
+    )
+    additional_contacts = models.TextField(
+        _("Additional contacts"),
+        max_length=400,
+        blank=True,
+        null=True,
+        help_text="Here you can store additional contacts, like Skype.",
+    )
+    updated_at = models.DateTimeField(
+        _("Last update"),
+        auto_now=True,
+        editable=False,
+    )
+    created_at = models.DateTimeField(
+        _("Created at"),
+        auto_now_add=True,
+        editable=False,
+    )
+
+    @property
+    def age(self) -> int:
+        """Return age of the candidate"""
+        if not self.date_of_birth:
+            return 0
+
+        from datetime import date
+
+        today = date.today()
+        month_day = (self.date_of_birth.month, self.date_of_birth.day)
+        return today.year - self.date_of_birth.year - ((today.month, today.day) < month_day)
```

Comment:
Maybe this code is simpler?
`return int((today - self.date_of_birth).days / 365.25)`

Reply:
Its simpler, but the proposed method with "365.25" in the birthday in certain years may fail and give the wrong age.

![image](https://user-images.githubusercontent.com/96374749/186652413-5a1e7de8-432b-4461-a375-4c0514bbc506.png)